### ISSUE TITLE
upgrade to latest standard release of sbt-bintray

### ISSUE THREAD

**Issue by SethTisue** [akka/akka-http#1355]
currently the build has:

```
project/plugins.sbt:addSbtPlugin("me.lessis" % "bintray-sbt" % "0.3.0-8-g6d0c3f8")
```

where this is a custom build from @2m 

could this be changed to use `"org.foundweekends" % "sbt-bintray" % "0.5.1"`? that's the latest version, and it's dbuild & Scala community friendly, so if the upgrade were done, we would no longer need to fork akka-http for the community build

@2m I'm not sure why the custom build was introduced in the first place...? the relevant commit is 45bc1d6049f82fa25f59c945aed2c61bd6d91433

**Comment by raboof** [comment]
If I recall correctly the custom build was created to provide bintray credentials in a secure way when publishing from Travis.

This is now no longer needed since https://github.com/sbt/sbt-bintray/pull/113 was merged, so indeed it'd be good to get back to mainline sbt-bintray again!

**Comment by 2m** [comment]
Fixed in #1355

**Comment by SethTisue** [comment]
thanks Martynas!
